MAUI/Picker/dealing-with-header-and-footer.md

```diff
@@ -0,0 +1,314 @@
+                xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+                x:Class="Picker_29.MainPage">
+
+        <sfPicker:SfPicker>
+            <sfPicker:SfPicker.FooterView>
+                <sfPicker:PickerFooterView 
+                    ShowOkButton="False" 
+                    OkButtonText="Done" 
+                    CancelButtonText="Exit"/>
+            </sfPicker:SfPicker.FooterView>
+        </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    using Syncfusion.Maui.Picker;
+    namespace Picker_29;
+
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfPicker picker = new SfPicker();
+            picker.FooterView.OkButtonShow = false;
+            picker.FooterView.OkButtonText= "Done";
+            picker.FooterView.CancelButtonText= "Exit";
+        }
+    }
+  
+{% endhighlight %}
+{% endtabs %}
+
+### Footer text style
+
+The .NET MAUI Picker control, Footer text style such as text color, font size, font family, font attributes can be customize as shown in the following code.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+                x:Class="Picker_29.MainPage">
+
+        <sfPicker:SfPicker>
+        <sfPicker:SfPicker.FooterView>
+          <sfPicker:PickerFooterView>
+            <sfPicker:PickerFooterView.TextStyle>
+              <sfPicker:PickerTextStyle TextColor="Gray" FontSize="18" FontAttributes="Italic"/>
+            </sfPicker:PickerFooterView.TextStyle>
+          </sfPicker:PickerFooterView>
+        </sfPicker:SfPicker.FooterView>
+        </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    using Syncfusion.Maui.Picker;
+    namespace Picker_29;
+
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfPicker picker = new SfPicker();
+            picker.FooterView.TextStyle = new PickerTextStyle()
+            {
+                TextColor = Colors.Gray,
+                FontSize = 18,
+                FontAttributes = FontAttributes.Italic
+            };
+        }
+    }
+  
+{% endhighlight %}
+{% endtabs %}
+
+## Perform validation with default validation button
+
+Picker allows performing validation based on `OK` or `Cancel` button by hooking SfPicker.OkButtonClicked and SfPicker.CancelButtonClicked. In this event, from the SelectionChangedEvent argument, current selected items can be obtained.
```

Comment:
Picker allows validation based on the `OK` or `Cancel` button by hooking into the SfPicker.OkButtonClicked and SfPicker.CancelButtonClicked. In this event, from the SelectionChangedEvent argument, current selected items can be obtained.

Reply:
Content modified